package.json

```diff
@@ -10,6 +10,8 @@
     "compile": "lerna exec -- tsc --project tsconfig.cjs.json",
     "compile:esm": "lerna exec --no-private -- tsc --project tsconfig.esm.json",
     "format": "prettier --write '**/*.{js,json,md,ts,tsx,yml}'",
+    "generate-config": "rcgen --verbose --force",
```

Comment:
I will probably add the `generate-api-docs` script soon. Should this be `generate:config` and `generate:api-docs` instead?